Code/Source/Camera/CameraSensor.cpp

```diff
@@ -188,10 +192,10 @@ namespace ROS2
                 AZ_Assert(Internal::FormatMappings.contains(format), "Unknown format in result %u", static_cast<uint32_t>(format));
                 sensor_msgs::msg::Image message;
                 message.encoding = Internal::FormatMappings.at(format);
+
                 message.width = descriptor.m_size.m_width;
                 message.height = descriptor.m_size.m_height;
-                message.step = message.width * sensor_msgs::image_encodings::bitDepth(message.encoding) / 8 *
-                    sensor_msgs::image_encodings::numChannels(message.encoding);
+                message.step = message.width * Internal::BitDepth.at(format);
```

Comment:
unrelated?

Reply:
I disagree - it is related to the elimination of the usage of exception, what is mentioned in [style](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#c-exceptions). With this change I do not need to include `sensor_msgs/image_encodings.hpp` that uses exception in header's code